package.json

```diff
@@ -6,9 +6,9 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "build": "tsc -d",
-    "build:watch": "npm run build -w",
+    "build:watch": "yarn build -w",
```

Comment:
watch requires yarn. devs have option to just build if they don't want to use watch

Reply:
I'm generally opposed to having `npm` and `yarn` co-exist in the same directory so if we require yarn we should remove package-lock and replace it with yarn.lock. 

Additionally, you can just run `yarn build -w` and the `-w` will be passed to the `tsc -d` command and will watch so it might be unnecessary to have the `yarn` command in the package.json itself.